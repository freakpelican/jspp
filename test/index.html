<html>
  <head>
    <title>Test page</title>
  </head>
  
  <body>
    <div class="test1">
      <?jspp console.log(document.title); end(); ?>
    </div>
    <div class="test2">
      <script type="application/jspp">
        require("request");
        $(this).text(require("./module").exp); 
        // end();
      </script>
    </div>
    
    <script>
      console.log("This should not show up!");
    </script>
  </body>
  
</html>