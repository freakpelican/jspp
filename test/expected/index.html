<html>
  <head>
    <title>Test page</title>
  </head>
  
  <body>
    <div class="test1">
      
    </div>
    <div class="test2">testing modules.... 1,2,3</div>
    
    <script>
      console.log("This should not show up!");
    </script>
  </body>
  
</html>